browser/importer/brave_importer_p3a.cc

```diff
@@ -21,6 +21,7 @@ enum class ImporterSource {
   kIE,
   kEdge,
   kEdgeChromium,
+  kVivaldi,
```

Comment:
I'm not 100% sure - but I think this might cause a problem. @iefremov buckets are limited to 8 values IIRC (for P3A), right? Is there a way we can properly handle sources/attribution for importing?

Reply:
(just mentioning myself to watch for the reply, thanks for the heads up @bsclifton )

Reply:
correct, we only have 8 buckets. Perhaps you could consider merging kEdge and kEdgeChromium, or removing IE? The thing is that we also should keep buckets reasonably large, I guess the current set of options could be deanonimising due to really small IE/kEdge cohorts.

 Or actually think about whether we need this metric at all, not sure how useful it is for us.

Reply:
I think if we change the enum, we'll need to version the question (so that old responses don't look like they're referencing new enum values)

Will think on this a bit

Reply:
Updated in https://github.com/brave/brave-core/pull/13990/commits/ed48255ddf99206bfb7618bebebb4ac798a63a90

Basically narrowed it down to 8 buckets and reviewed with @rebron. Vivaldi seems to have a lower MAU than Opera (at least from public numbers) so I kept Opera distinct and we can club Vivaldi under `kOther`. Bookmarks import is significant enough to not club it under `kOther`